pom.xml

```diff
@@ -43,25 +46,36 @@
     </developer>
   </developers>
 
+  <repositories> 
+    <repository>
+      <id>maven.snapshots</id>
+      <name>Maven Central Snapshot Repository</name>
+      <url>https://oss.sonatype.org/content/repositories/snapshots/</url>
+      <releases>
```

Comment:
How to publish to sonatype snapshots? I need to publish this snapshot version of azure-functions-java-library